lib/kino_vega_lite/chart_cell.ex

```diff
@@ -412,7 +412,8 @@ defmodule KinoVegaLite.ChartCell do
   defp columns_for(data) do
     with true <- implements?(Table.Reader, data),
          data = {_, %{columns: columns}, _} <- Table.Reader.init(data),
```

Comment:
If we are allowing empty datasets, then let's check for non-empty columns.

```suggestion
         data = {_, %{columns: [_ | _] = columns}, _} <- Table.Reader.init(data),
```